.github/workflows/coverage.yml

```
@@ -39,4 +39,6 @@ jobs:
       - name: Test NetworkX
         run: |
           pytest --cov=networkx --runslow --doctest-modules --durations=20 --pyargs networkx
-          codecov
+
+      - name: Updload to codecov
```

This is passing tests.  How do find out if we need a token? Merge this and see what happens? :}
```suggestion
      - name: Upload to codecov
```